ansible_base/lib/utils/response.py

```diff
@@ -4,6 +4,7 @@
 from typing import Optional, Sequence
 from urllib.parse import urlparse
 
+from crum import get_current_request
```

Comment:
Talking it out loud - this import is okay because django-crum is in the DAB main requirements?

Reply:
Yes, we list `django-crum` in requirements.in